static/src/stylesheets/module/_discussion.scss

```diff
@@ -903,6 +915,55 @@ $comment-recommend-button-size: 19px;
     }
 }
 
+.sharing-component {
+    height: 1.2rem;
+    vertical-align: middle;
+    margin-left: 5px;
+
+    &:hover {
+        .sharing-buttons {
+            opacity: 1;
+            margin-left: 5px;
+        }
+
+        .comment-facebook-icon,
+        .comment-twitter-icon {
+            height: 1.2rem;
+            min-width: 1.2rem;
+            margin: 0;
+            cursor: pointer;
+        }
+    }
+}
+
+.sharing-text {
+    cursor: pointer;
+    display: inline-block;
+    float: left;
+}
+
+.comment-share-icon {
+    fill: $guardian-brand;
+}
+
+.comment-share-icon svg {
+    vertical-align: text-top;
+}
+
+.sharing-buttons {
+    opacity: 0;
+    cursor: pointer;
+    display: inline-block;
+}
+
+.comment-facebook-icon,
+.comment-twitter-icon {
+    height: 0;
+    min-width: 0;
+    margin: .6rem;
+    transition: all .5s ease-in-out;
```

Comment:
This is only working on hover in, but not on hover out. Can anyone help me please?